### PR TITLE
Bump version: 1.3.1 → 1.4.0

### PR DESCRIPTION
**Pull request by mr-cal**
- [X] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,27 @@
 Changelog
 *********
 
+
+1.4.0 (2022-08-22)
+------------------
```
Comment (facundobatista) [inline docs/changelog.rst:7]: What about `temporarily_pull_file`?

**Review by facundobatista — CHANGES_REQUESTED** [review]
Missing changelog line?

**Review by cmatsuoka — APPROVED** [review]
If Colin is aware, ship it!

File: docs/changelog.rst (hunk 1 above)
Comment (mr-cal) [reply docs/changelog.rst:7]: Good catch, I added that back into the changelog.

**Review by facundobatista — APPROVED** [review]
+1

**Pull request merged**